src/fidesops/ops/api/v1/scope_registry.py

```diff
@@ -20,9 +20,11 @@
 PRIVACY_REQUEST_READ = "privacy-request:read"
 PRIVACY_REQUEST_DELETE = "privacy-request:delete"
 PRIVACY_REQUEST_CALLBACK_RESUME = (
-    "privacy-request:resume"  # User has permission to restart a paused privacy request
+    "privacy-request:resume"  # User has permission to resume a privacy request
 )
 PRIVACY_REQUEST_REVIEW = "privacy-request:review"
+PRIVACY_REQUEST_UPLOAD_DATA = "privacy-request:upload_data"
+PRIVACY_REQUEST_VIEW_DATA = "privacy-request:view_data"
```

Comment:
do we also need to add new scopes in fideslib at this point?

Reply:
@sanders41 are we keeping fideslib scopes synced with fidesops scopes with the fides unification work, or is it okay that this is the most current list?

Reply:
I have been trying to keep them in sync, but it won't break if they get out of sync for now.